Assets/Scripts/PlayerMovement.cs

```diff
@@ -30,43 +39,83 @@ void Update()
 
             transform.position += (targetPosition - startPosition) * speed * Time.deltaTime;
             return;
-        }
+        }//should the rest not go under an else here?
```

Comment:
An else here would be redundant, since the if statement ends with a return, it'll never execute the lower code if the player is moving.

If you think the code would be cleaner if we ditch the return and add an else though, feel free to do that.